src/app/fyle/split-expense/split-expense.page.ts

```
@@ -861,108 +863,116 @@
+        }
 
-    this.isCorporateCardsEnabled$ = orgSettings$.pipe(
-      map(
-        (orgSettings) =>
-          orgSettings.corporate_credit_card_settings && orgSettings.corporate_credit_card_settings.enabled
-      )
-    );
+        this.isCorporateCardsEnabled$ = orgSettings$.pipe(
+          map(
+            (orgSettings) =>
+              orgSettings.corporate_credit_card_settings && orgSettings.corporate_credit_card_settings.enabled
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)